cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/Users/binh/Documents/workspace/lena/.waf-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Tools/c_config.py", line 459, in run_c_code\n    bld.compile()\n  File "/Users/binh/Documents/workspace/lena/.waf-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 4339275536: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'test.cpp.1.o\', \'-o\', \'/Users/binh/Documents/workspace/lena/build/.conf_check_5ef57bc0d8aa3565949b6ce1186fd4a7/testbuild/testprog\', \'-lrt\']\n'
